lib/maths/COutliers.cc

```diff
@@ -33,7 +33,7 @@ using namespace outliers_detail;
 
 namespace {
 
-const std::string COMPUTE_OUTLIER_SCORES{"compute_outlier_scores"};
+const std::string COMPUTE_OUTLIERS{"computing_outliers"};
```

Comment:
if we decide to match labels with content we should do it here too

Reply:
See [3d0b42e](https://github.com/elastic/ml-cpp/pull/1179/commits/3d0b42e404747cdfc195c609f369118de806fafc).